KS-2-1 averageScore

CasTeTRU opened the pull request.

CasTeTRU committed:
KS-2-1 averageScore

CasTeTRU commented:
KS-2-1 averageScore
In this case there is two digits after comma, should I be more specific and use output formatting in case of more than two digits after comma to match the requirements of the task?

ievetrovdev commented on src/main/kotlin/lesson_2/lesson2_task1.kt:
```diff
@@ -0,0 +1,6 @@
+package org.example.lesson_2
+
+fun main() {
+    val averageScore: Double = (3.0 + 4.0 + 3.0 + 5.0) / 4
```
Значения в этом коде являются [магическими числами](https://ru.wikipedia.org/wiki/%D0%9C%D0%B0%D0%B3%D0%B8%D1%87%D0%B5%D1%81%D0%BA%D0%BE%D0%B5_%D1%87%D0%B8%D1%81%D0%BB%D0%BE_(%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5)#%D0%9F%D0%BB%D0%BE%D1%85%D0%B0%D1%8F_%D0%BF%D1%80%D0%B0%D0%BA%D1%82%D0%B8%D0%BA%D0%B0_%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F). Не лучшая практика. Предлагаю добавить переменные)

ievetrovdev commented:
> KS-2-1 averageScore In this case there is two digits after comma, should I be more specific and use output formatting in case of more than two digits after comma to match the requirements of the task?

В целом это пока довольно базовые задачи. Продвинутыми способами решать не запрещается, но и простое решение, как у тебя в данной задаче корректно.

ievetrovdev reviewed (changes requested):
И давай вопросы писать на русском, мы все таки не иностранная компания)

CasTeTRU committed:
KS-2-1-Average-Score
Убрал магические числа добавив новые переменные

ievetrovdev commented on src/main/kotlin/lesson_2/lesson2_task1.kt:
```diff
@@ -0,0 +1,12 @@
+    val secondPupilScore = 4.0
+    val thirdPupilScore = 3.0
+    val fourthPupilScore = 5.0
+    val numberOfPupils = 4
```
Для красоты и читаемости лучше разбить код на блоки, разделяемые пустыми строками. Примерно так:
 - объявление переменных
 - вычисления
 - вывод результата

CasTeTRU committed:
KS-2-1-averageScore
Добавлено деление по блокам: обьявление переменных, вычисления, вывод результата

ievetrovdev reviewed (approved):
Теперь все корректно, аппрув!

И на будущее, предлагаю не резолвить треды здесь, все равно мне приходится их раскрывать чтобы войти в контекст)
